package.json

```diff
@@ -24,6 +24,9 @@
   "bugs": {
     "url": "https://github.com/JaKXz/stylelint-webpack-plugin/issues"
   },
+  "engines": {
+    "node": ">=4.0.0"
```

Comment:
while `.travis.yml` does not test against Node < 4, I feel as though I might as well have kept it there since we're not using any language features. Based on my comment above, I think we could keep it that way.

And if we can't for some reason, I'd rather use the `object-assign` ponyfill for now.